packages/one-app-bundler/webpack/loaders/externals-loader.js

```diff
@@ -12,19 +12,59 @@
  * under the License.
  */
 
+const path = require('path');
 const loaderUtils = require('loader-utils');
+const readPkgUp = require('read-pkg-up');
+const babel = require('@babel/core');
 
-function requiredExternalsLoader() {
-  const options = loaderUtils.getOptions(this);
+const packageRoot = process.cwd();
+const { packageJson } = readPkgUp.sync();
+
+function externalsLoader(content) {
+  const { externalName, bundleTarget } = loaderUtils.getOptions(this);
+  if (bundleTarget === 'server') {
```

Comment:
Should use the constant here
`import { BUNDLE_TYPES } from '../constants/enums.js';`
```suggestion
  if (bundleTarget === BUNDLE_TYPES.SERVER) {
```

Reply:
we don't have these type of constants in `one-app-bundler`